docs/running-on-kubernetes.md

```diff
@@ -200,42 +200,88 @@ from the other deployment modes. See the [configuration page](configuration.html
   </td>
 </tr>
 <tr>
-  <td><code>spark.kubernetes.submit.caCertFile</code></td>
+  <td><code>spark.kubernetes.authentication.submission.caCertFile</code></td>
```

Comment:
this key is kinda long -- thoughts on shortening `authentication` to just `auth` ?